static/app/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/releaseSearchBar.tsx

```diff
@@ -75,7 +81,26 @@ export function ReleaseSearchBar({
     return acc;
   }, {});
 
-  return (
+  return organization.features.includes('search-query-builder-releases') ? (
+    <SearchQueryBuilder
+      initialQuery={widgetQuery.conditions}
+      filterKeySections={filterKeySections}
+      filterKeys={supportedTags}
```

Comment:
Could you extract `supportedTags` outside of the component so that the reference doesn't change on rerender? The new component is pretty heavy so we want to avoid it where possible